test/test.ml

```diff
@@ -416,8 +437,11 @@ let test_renderers () =
           log " Render target supported: %b" (Sdl.render_target_supported r);
           let rect = Sdl.Rect.create 20 30 20 20 in
           assert (Sdl.render_set_clip_rect r (Some rect) = Ok ());
+          assert( Sdl.render_is_clip_enabled r);
```

Comment:
Style. Could you please add a space before the paren and remove the one after it.